src/igvfd/audit/biosample.py

```diff
@@ -0,0 +1,20 @@
+from snovault.auditor import (
+    audit_checker,
+    AuditFailure,
+)
+from .formatter import (
+    audit_link,
+    path_to_text,
+)
+
+
+@audit_checker('Biosample', frame='object')
+def audit_biosample_nih_institutional_certification(value, system):
+    '''Biosample objects must specify an NIH Institutional Certification required for human data.'''
+    if ('nih_institutional_certification' not in value) and (value.get('taxa') == 'Homo sapiens'):
+        value_id = system.get('path')
```

Comment:
why not use "sample_id"? it would be more clear this way ID of what you are looking at. And why use system.get('path'), instead of `value.get('@id')`

Reply:
Done